src/components/loginform.jsx

```diff
@@ -14,11 +17,10 @@ export function LoginForm({ onLoginSuccess }) {
 
   // Efecto para cargar las credenciales almacenadas en localStorage
   useEffect(() => {
-    const storedCredentials = localStorage.getItem('credentials');
-    
-    if (storedCredentials) {
-      const parsedCredentials = JSON.parse(storedCredentials);
-      setCredentials({ email: parsedCredentials.username, password: parsedCredentials.password });
+    const { username, password } = readFromLocalStorage();
```

Comment:
suggestion: `services/local-storage.js` exporting functions realted to local storage, this way those would be easy to test isolated and can control better possible exceptions